src/pages/Registration/Registration.tsx

```diff
@@ -19,6 +25,20 @@ export default function SignIn() {
 	const [email, setEmail] = useState('');
 	const [password, setPassword] = useState('');
 	const [isCognitoLoading, setIsCognitoLoading] = useState<boolean>(false);
+	const navigate = useNavigate();
+	const queryClient = useQueryClient();
+	const userError = useSnackbarOnError();
+
+	const { mutate: registerUser } = useMutation(
+		[entities.user],
+		(user: RegisterUserRequest) => {
+			return UserService.register(user);
+		},
+		{
+			onError: useSnackbarOnError(),
+			onSuccess: () => queryClient.invalidateQueries(entities.user),
```

Comment:
we don't need invalidateQueries because we won't get it

Reply:
Deleted